controllers/admin/AdminCountriesController.php

```diff
@@ -478,14 +479,6 @@ protected function processBulkAffectZone()
 
     protected function displayValidFields()
     {
-        /* The following translations are needed later - don't remove the comments!
```

Comment:
Well, this line tells you to NOT remove the comments 😛  so I think unless we are 100% sure it's OK to remove it, better keep it. We have some magic related to translation scanning that might need this.

Reply:
:+1:

Reply:
Thanks for changing !

Maybe, to be sure, it works with `[]` we should run the catalog export process 🤔  ?